web/react/stores/preference_store.jsx

```diff
@@ -122,5 +122,6 @@ class PreferenceStoreClass extends EventEmitter {
 }
 
 const PreferenceStore = new PreferenceStoreClass();
+PreferenceStore.setMaxListeners(0);
```

Comment:
Actually, each post already subscribes to UserStore per-post so I think it's not a bad design or something to worry about( and it has maxlisteners 0 too). That way it wouldnt render every post from scratch.